cmd/rpcdaemon/commands/send_transaction_test.go

```diff
@@ -27,33 +27,33 @@ import (
 
 func TestSendRawTransaction(t *testing.T) {
 	t.Skip("Flaky test")
-	m, require := stages.Mock(t), require.New(t)
+	m, assertions := stages.Mock(t), require.New(t)
```

Comment:
This is recommendations of libs author to do: 
require := require.New()
and 
assert := assert.New()

https://github.com/stretchr/testify#assert-package

also in this package “require” and “assert” have different meaning, don’t need to mix them

Reply:
Oh okay, I used `assertions`, because the `require.New()` method returned an `Assertion` type.
I'll change it, Thanks